v2/support/modal_sample_app/assets/iframe.html

```diff
@@ -0,0 +1,88 @@
+<html>
+<head>
+  <link rel="stylesheet" href="styles.css" type="text/css">
+</head>
+<body>
+  <section data-main>Loading...</section>
+  <!-- https://github.com/zendesk/zendesk_app_framework_sdk -->
+  <script type="text/javascript" src="https://assets.zendesk.com/apps/sdk/2.0/zaf_sdk.js"></script>
+  <script>
+    // Initialise the Zendesk JavaScript API client
+    // https://developer.zendesk.com/apps/docs/apps-v2
+    var client = ZAFClient.init();
+
+    function init(location) {
+      location === 'modal' ? new ModalApp() : new TicketApp();
```

Comment:
Yea, just not sure it will make sense for app developers why we have empty constructors and `init` methods.